tests/bgp/test_bgp_session_flap.py

```diff
@@ -90,10 +90,9 @@ def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_
         'namespace': namespace
     }
 
-    logger.info("DUT BGP Config: {}".format(duthost.shell("vtysh -n {} -c \"show run bgp\"".format(namespace),
+    logger.info("DUT BGP Config: {}".format(duthost.shell("vtysh -n {} -c \"show run bgp\"".format(asic_index),
```

Comment:
the vtysh command may not work on single asic linecards/devices

Reply:
Thanks @arlakshm. Fixed it now. Please reapprove.